mne_bids/read.py

```diff
@@ -253,6 +277,13 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
     if channels_fname is not None:
         raw = _handle_channels_reading(channels_fname, bids_fname, raw)
 
+    # Try to find an associated sidecar.json to get information about the
+    # recording snapshot
+    sidecar_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                           f"{kind}.json", allow_fail=True)
```

Comment:
so far we have not done any f-string formatting, because it was introduced in Python 3.6 ... however, we are also covering Python 3.5 with this package.